src/common/settings_objects.cpp

```diff
@@ -277,27 +277,27 @@ namespace PowerToysSettings
         return L"RESOURCE ID NOT FOUND: " + std::to_wstring(resource_id);
     }
 
-    PowerToyValues::PowerToyValues(std::wstring_view powertoy_name)
+    PowerToyValues::PowerToyValues(std::wstring_view powertoy_name, std::wstring_view powertoy_key)
     {
-        _name = powertoy_name;
+        _key = powertoy_key;
         set_version();
         m_json.SetNamedValue(L"name", json::value(powertoy_name));
```

Comment:
I think this should be modified as well. The name of the module within the settings.json file is not user facing and is only used internally by the codebase. Settingsv2 does not have this value localized. It uses the module name in english.

Reply:
The name of the module is displayed by settings v1. To check this, change the name `LAUNCHER_NAME` variable in resx file, delete the localized files, and start runner with settings v1.

Reply:
Yes, that is correct, However, I think there would be an issue when we upgrade to settingsV2. 
If the settings.json file exists within say the FancyZones folder, with the following value ` name:FancyZones-in-french, version ...` then when settingsV2 loads, it loads the properties from the settings.json file where the name is localized.
then when it tries to send the information to the runner, it just converts the json object to string where the name of the module is localized. This way the settings UI changes that the user makes won't reflect within the powertoy.

Reply:
I just validated it and seems like it uses the moduleName itself so this should not be an issue.

Reply:
I think we currently don't face any issues because the `name` value within each module-specific json file is not used anywhere. I think we should add some comments to call that out specifically that the `name` attribute in the json file is the localized name.

Reply:
Created issue https://github.com/microsoft/PowerToys/issues/7384 to track this.